operator/templates/backup-job-rbac.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if eq .Params.BACKUP_RESTORE_ENABLED "true" }}
```

Comment:
can this be a toggle task? when we disable `BACKUP_RESTORE_ENABLED ` using `false` these resources won't be cleaned up